frontend/public/src/components/CourseProductDetailEnroll.js

```diff
@@ -248,9 +381,19 @@ export class CourseProductDetailEnroll extends React.Component<
             </div>
 
             <div className="row certificate-pricing-row">
-              <div className="col-6 certificate-pricing">
+              <div className="col-6 certificate-pricing d-flex align-items-center">
+                <div className="certificate-pricing-logo">
+                  <svg viewBox="0 0 40 44" xmlns="http://www.w3.org/2000/svg">
+                    <path
+                      fillRule="evenodd"
+                      clipRule="evenodd"
+                      d="M33.1301 23.314L39.8422 34.7838C40.1671 35.3396 39.9743 36.0504 39.4115 36.3713C39.2329 36.4731 39.0303 36.5269 38.8241 36.527L31.4033 36.6337L27.6205 42.9098C27.5701 42.9923 27.5099 43.0684 27.4413 43.1368C27.2159 43.3603 26.9102 43.4868 26.5909 43.4887C26.5404 43.4887 26.4921 43.484 26.4439 43.4796C26.0796 43.4373 25.756 43.23 25.5683 42.9189L20.0007 33.3956L14.4279 42.9189C14.2471 43.2273 13.9335 43.4372 13.5764 43.4887C13.5208 43.4957 13.4648 43.4996 13.4087 43.5C13.1067 43.4999 12.8165 43.3852 12.5974 43.18C12.5117 43.0995 12.4364 43.0087 12.3733 42.9098L8.59171 36.6337L1.17668 36.527C0.756404 36.5209 0.370025 36.2979 0.158594 35.9391C-0.0503957 35.5823 -0.052981 35.143 0.151699 34.7838L6.86076 23.3192C5.49809 21.0597 4.70092 18.429 4.70092 15.6081C4.70092 7.26415 11.5494 0.5 19.9975 0.5C28.4456 0.5 35.2941 7.26415 35.2941 15.6081C35.2908 18.4271 34.4926 21.0559 33.1301 23.314ZM10.2775 34.8859L13.388 40.0475L18.8818 30.6606C14.6737 30.3598 10.9406 28.3867 8.3615 25.4002L3.19336 34.2322L9.2835 34.3184C9.69214 34.3244 10.0683 34.5392 10.2775 34.8859ZM19.9972 28.392C15.4671 28.392 11.4865 26.0893 9.17387 22.6084C9.17309 22.6075 9.17243 22.6064 9.17182 22.6054C9.17134 22.6046 9.17088 22.6037 9.17043 22.6028C9.16945 22.601 9.16845 22.5991 9.16712 22.5974C7.83504 20.5879 7.05396 18.1895 7.05396 15.6082C7.05396 8.54801 12.8489 2.82445 19.9972 2.82445C27.1421 2.83282 32.932 8.55141 32.9405 15.6082C32.9405 18.1908 32.1588 20.5901 30.8256 22.6002L30.8253 22.6008C28.5135 26.0858 24.5306 28.392 19.9972 28.392ZM29.7162 34.8857C29.9258 34.5389 30.3024 34.3242 30.7113 34.3183L36.8015 34.232L31.6302 25.3931C29.1549 28.2601 25.6209 30.2062 21.6202 30.6274L21.3601 31.0721L26.6068 40.0474L29.7162 34.8857Z"
+                      fill="#6F7175"
+                    />
```

Comment:
Nit: this should probably be extracted into a file-level `const` definition since it clutters the function a tad.